images/build_images

```diff
@@ -54,43 +63,63 @@ def main():
     parser.add_argument('-p', '--push', help='Push Docker images after building', action='store_true')
     args = parser.parse_args()
 
-    image_folder = Path(Path(__file__).parent, 'cloudera_service').resolve()
-
     if args.dry_run:
         logger.info('Doing dry-run of tool ...')
 
     sdc_version = _get_sdc_version(PARCEL_MANIFEST_URL_TEMPLATE.format(build=args.sdc_build,
                                                                        s3_bucket=args.sdc_s3_bucket))
+
+    images = _get_sdc_images(args, sdc_version)
+    _build_push_images(args, images, sdc_version)
+
+
+def _build_push_images(args, images, sdc_version):
+    image_folder = Path(Path(__file__).parent, 'cloudera_service').resolve()
+    legacy_libs_index = _get_legacy_stage_libs_index(sdc_version, args.sdc_build, args.sdc_s3_bucket)
+    csd_url = CSD_URL_TEMPLATE.format(sdc_version, build=args.sdc_build, s3_bucket=args.sdc_s3_bucket)
+
+    for image in images:
+        cmd_elements = ['docker build -t {}'.format(image.name),
+                        '--label "legacy-libs={}"'.format(legacy_libs_index),
+                        '--build-arg CSD_URL={}'.format(csd_url),
+                        '--build-arg PARCEL_URL={}'.format(image.parcel_url),
+                        '--build-arg PRODUCT={}'.format(SDC_PRODUCT_NAME),
+                        str(image_folder)]
+        cmd = ' '.join(cmd_elements)
```

Comment:
Just as a thought, do we need to have `cmd_elements` be a list of can we just have strings concatenated with ( ) like you've done earlier in the review?

Reply:
Done.